File: SteamDroid/src/com/flyingpie/steamdroid/app/Preferences.java
package com.flyingpie.steamdroid.app;

import android.os.Bundle;
import android.preference.PreferenceActivity;

import com.flyingpie.steamdroid.R;

public class Preferences extends PreferenceActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		addPreferencesFromResource(R.layout.preferences);
	}	
}

File: SteamDroid2/SteamDroid2/App/Preferences.cs
using Android.App;
using Android.OS;
using Android.Preferences;

namespace SteamDroid2.App
{
	[Activity (Label = "Preferences")]			
	public class Preferences : PreferenceActivity
	{
		protected override void OnCreate(Bundle bundle)
		{
			base.OnCreate(bundle);
			AddPreferencesFromResource(Resource.Layout.Preferences);
		}
	}
}

